### PR TITLE
Test on Python37

### PR DESCRIPTION
**Pull request by max-sixty**
What do you think about skipping conda tests for the moment @tswast ? 

The test on 3.7 seems less controversial (probably should split these up, but both small PRs)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,12 +8,13 @@ matrix:
   - os: linux
     python: 3.5
     env: PYTHON=3.5 PANDAS=0.18.1 COVERAGE='true' LINT='false'
+  # TODO: fix conda issues around namespace packages
```
Comment (tswast) [inline .travis.yml]: Please link to https://github.com/pydata/pandas-gbq/issues/189 in the comment.

**Comment by max-sixty** [comment]
@tswast let me know if you recognize these ADC errors in the 3.7 build: https://travis-ci.org/max-sixty/pandas-gbq/jobs/422752133

Could they be related to us using sudo (which is required for 3.7 it seems)?

**Comment by tswast** [comment]
> http://metadata.google.internal/computeMetadata/v1/instance/service-accounts/default/?recursive=true from the Google Compute Enginemetadata service. Status: 404 Response

I think this is related to the fact that Travis CI runs on GCE but it locks down access to APIs. Maybe `google-auth` doesn't account for this case?

**Comment by tswast** [comment]
Python 3.7 builds added in https://github.com/pydata/pandas-gbq/pull/232

**Pull request closed without merging**